gh-pr-context.py

```
@@ -121,6 +205,35 @@ def resolve_pr_number():
     return val
 
 
+def _detect_fork_parent(owner_repo):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_detect_fork_parent`

(ANN202)